.github/workflows/snap.yml

```diff
@@ -0,0 +1,23 @@
+name: Pebble snap
+
+on: [pull_request]
```

Comment:
It seems like this should happen on push to master and on new release, rather than on pull request. Maybe this?

```
on:
  push:
    branches: [master]
  release:
    types: [published]
```

Reply:
this specific action, atm, is purely a test for making sure the snapcraft.yaml is sane and that the snap can be successfully built. after all, we shouldn't merge changes if those are going to make the store builds fail.

the build on `push: master` will happen from the store, and on `release` will be implemented separately later on (see "Next Steps" above)